src/coreclr-debug/install.ts

```diff
@@ -0,0 +1,226 @@
+                    // build that I have tested.
+                    this._util.log("Error: The .NET CLI did not correctly restore debugger files. Ensure that you have .NET CLI version 1.0.0 build #001584 or newer. You can check your .NET CLI version using 'dotnet --version'.");
+                    return reject("The .NET CLI did not correctly restore debugger files.");
+                }
+            });
+        });
+    }
+
+    private writeProjectJson(runtimeId: string): Promise<void> {
+        return new Promise<void>((resolve, reject) => {
+            const projectJsonPath = path.join(this._util.coreClrDebugDir(), 'project.json');
+            this._util.log('Creating ' + projectJsonPath);
+
+            const projectJson = this.createProjectJson(runtimeId);
+
+            fs.writeFile(projectJsonPath, JSON.stringify(projectJson, null, 2), {encoding: 'utf8'}, (err) => {
+                if (err) {
+                    this._util.log('Error: Unable to write to project.json: ' + err.message);
+                    reject(err.code);
+                }
+                else {
+                    resolve();
+                }
+            });
+        });
+    }
+
+    private createProjectJson(targetRuntime: string): any
+    {
+        let projectJson = {
+            name: "dummy",
+            buildOptions: {
+                emitEntryPoint: true
+            },
+            dependencies: {
+                "Microsoft.VisualStudio.clrdbg": "14.0.25406-preview-3044032",
+                "Microsoft.VisualStudio.clrdbg.MIEngine": "14.0.30606-preview-1",
+                "Microsoft.VisualStudio.OpenDebugAD7": "1.0.20527-preview-1",
+                "NETStandard.Library": "1.5.0-rc2-24027",
+                "Newtonsoft.Json": "7.0.1",
+                "Microsoft.VisualStudio.Debugger.Interop.Portable": "1.0.1",
+                "System.Collections.Specialized":  "4.0.1-rc2-24027",
+                "System.Collections.Immutable": "1.2.0-rc2-24027", 
+                "System.Diagnostics.Process" : "4.1.0-rc2-24027",
+                "System.Diagnostics.StackTrace":  "4.0.1-rc2-24027",  
+                "System.Dynamic.Runtime": "4.0.11-rc2-24027",
+                "Microsoft.CSharp": "4.0.1-rc2-24027",
+                "System.Threading.Tasks.Dataflow": "4.6.0-rc2-24027",
+                "System.Threading.Thread": "4.0.0-rc2-24027", 
+                "System.Xml.XDocument": "4.0.11-rc2-24027",
+                "System.Xml.XmlDocument": "4.0.1-rc2-24027",  
+                "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
+                "System.ComponentModel":  "4.0.1-rc2-24027",  
+                "System.ComponentModel.Annotations":  "4.1.0-rc2-24027",  
+                "System.ComponentModel.EventBasedAsync":  "4.0.11-rc2-24027",
+                "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027",
+                "System.Net.Http":  "4.0.1-rc2-24027",
+                "Microsoft.NetCore.DotNetHostPolicy": "1.0.1-rc-002702"
```

Comment:
I don't know, but I since we are dynamically generating the file anyway, I am sure the simplest option is to only add the element if we are doing the offline install.